src/pages/home/report/ReportActionCompose.js

```diff
@@ -86,7 +86,7 @@ const propTypes = {
     isFocused: PropTypes.bool.isRequired,
 
     /** Is the composer full size */
-    isComposerFullSize: PropTypes.bool.isRequired,
+    isComposerFullSize: PropTypes.bool,
```

Comment:
Does this requirement need to be removed? I don't see you removing the prop `isComposerFullSize` anywhere so I'd assume it should stay the same?